src/react/ReportControls/components/ViewSettings/ViewSettings.tsx

```diff
@@ -35,18 +34,14 @@ const viewSettingsMap: Record<Exclude<ReportTypes, 'table'>, FC> = {
   due: ScatterPlotViewSettings,
 };
 
-const useReportType = () => {
-  return useCanObservable<ReportTypes>(value.from(routeData, 'primaryReportType'));
-};
-
 const ViewSettings: FC = () => {
-  const currentReportType = useReportType();
+  const [primaryReportType] = usePrimaryReportType();
 
-  if (currentReportType === 'table') {
+  if (primaryReportType !== 'start-due' && primaryReportType !== 'due') {
```

Comment:
I found out there's a 4th possible value for this ("start-due"), so instead of checking against "table" and "start-due", I refactored to check against the only 2 working values for this component